Jenkinsfile

```diff
@@ -58,7 +58,16 @@ node {
 onlyOnMaster {
     milestone()
     stage('qa deploy') {
-        deployApp(image: image, app: 'h', env: 'qa', region: "us-west-1")
+        parallel(
+            us_h: {
+                deployApp(image: image, app: "h", env: "qa", region: "us-west-1")
+            },
+            us_ws: {
+		// Workaround to ensure all parallel builds happen. See https://hypothes-is.slack.com/archives/CR3E3S7K8/p1625041642057400
+                sleep 2
+                deployApp(image: image, app: "h-websocket", env: "qa", region: "us-west-1")
+            }
+        )
```

Comment:
Introduced parallel builds to the `qa` stage.

2x `deployApp` calls are being made. The first is to deploy to the `h` environment as we always have, and the second takes care of the `h-websocket`.

I have moved the comment relating to the use of the `sleep 2` command from the `prod` section to `qa`. My thinking is you only need to read it once the first time the command is used.